Flaky test: TestIcebergMetadataListing.testTableColumnListing

See: https://github.com/prestodb/presto/actions/runs/12403254688/job/34626370903?pr=24091

Error stack:

```
[ERROR] com.facebook.presto.iceberg.TestIcebergMetadataListing.testTableColumnListing  Time elapsed: 1.146 s  <<< FAILURE!
java.lang.AssertionError: 
For query: 
 SELECT table_name, column_name FROM iceberg.information_schema.columns WHERE table_schema = 'test_schema'
 actual column types:
 [varchar, varchar]
expected column types:
 [varchar, varchar]

not equal
Actual rows (6 of 6 extra rows shown, 11 rows in total):
    [test_view2_renamed, _string]
    [test_view2_renamed, _integer]
    [iceberg_test_table, _string]
    [iceberg_test_table, _integer]
    [test_view_renamed, _string]
    [test_view_renamed, _integer]

	at org.testng.Assert.fail(Assert.java:110)
	at com.facebook.presto.tests.QueryAssertions.assertEqualsIgnoreOrder(QueryAssertions.java:280)
	at com.facebook.presto.tests.QueryAssertions.assertQuery(QueryAssertions.java:233)
	at com.facebook.presto.tests.QueryAssertions.assertQuery(QueryAssertions.java:106)
	at com.facebook.presto.tests.AbstractTestQueryFramework.assertQuery(AbstractTestQueryFramework.java:174)
	at com.facebook.presto.iceberg.TestIcebergMetadataListing.testTableColumnListing(TestIcebergMetadataListing.java:113)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
	at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
	at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
	at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
	at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
	at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:750)
```

--- a/presto-iceberg/src/test/java/com/facebook/presto/iceberg/TestIcebergMetadataListing.java
+++ b/presto-iceberg/src/test/java/com/facebook/presto/iceberg/TestIcebergMetadataListing.java
@@ -157,16 +157,18 @@ public class TestIcebergMetadataListing
     @Test
     public void testRenameView()
     {
-        assertQuerySucceeds("CREATE TABLE iceberg.test_schema.iceberg_test_table (_string VARCHAR, _integer INTEGER)");
-        assertUpdate("CREATE VIEW iceberg.test_schema.test_view_to_be_renamed AS SELECT * FROM iceberg.test_schema.iceberg_test_table");
-        assertUpdate("ALTER VIEW IF EXISTS iceberg.test_schema.test_view_to_be_renamed RENAME TO iceberg.test_schema.test_view_renamed");
-        assertUpdate("CREATE VIEW iceberg.test_schema.test_view2_to_be_renamed AS SELECT * FROM iceberg.test_schema.iceberg_test_table");
-        assertUpdate("ALTER VIEW iceberg.test_schema.test_view2_to_be_renamed RENAME TO iceberg.test_schema.test_view2_renamed");
-        assertQuerySucceeds("SELECT * FROM iceberg.test_schema.test_view_renamed");
-        assertQuerySucceeds("SELECT * FROM iceberg.test_schema.test_view2_renamed");
-        assertUpdate("DROP VIEW iceberg.test_schema.test_view_renamed");
-        assertUpdate("DROP VIEW iceberg.test_schema.test_view2_renamed");
-        assertUpdate("DROP TABLE iceberg.test_schema.iceberg_test_table");
+        assertQuerySucceeds("CREATE SCHEMA iceberg.test_rename_view_schema");
+        assertQuerySucceeds("CREATE TABLE iceberg.test_rename_view_schema.iceberg_test_table (_string VARCHAR, _integer INTEGER)");
+        assertUpdate("CREATE VIEW iceberg.test_rename_view_schema.test_view_to_be_renamed AS SELECT * FROM iceberg.test_rename_view_schema.iceberg_test_table");
+        assertUpdate("ALTER VIEW IF EXISTS iceberg.test_rename_view_schema.test_view_to_be_renamed RENAME TO iceberg.test_rename_view_schema.test_view_renamed");
+        assertUpdate("CREATE VIEW iceberg.test_rename_view_schema.test_view2_to_be_renamed AS SELECT * FROM iceberg.test_rename_view_schema.iceberg_test_table");
+        assertUpdate("ALTER VIEW iceberg.test_rename_view_schema.test_view2_to_be_renamed RENAME TO iceberg.test_rename_view_schema.test_view2_renamed");
+        assertQuerySucceeds("SELECT * FROM iceberg.test_rename_view_schema.test_view_renamed");
+        assertQuerySucceeds("SELECT * FROM iceberg.test_rename_view_schema.test_view2_renamed");
+        assertUpdate("DROP VIEW iceberg.test_rename_view_schema.test_view_renamed");
+        assertUpdate("DROP VIEW iceberg.test_rename_view_schema.test_view2_renamed");
+        assertUpdate("DROP TABLE iceberg.test_rename_view_schema.iceberg_test_table");
+        assertQuerySucceeds("DROP SCHEMA IF EXISTS iceberg.test_rename_view_schema");
     }
     @Test
     public void testRenameViewIfNotExists()

